src/Components/NavBar/NavItem.tsx

```diff
@@ -26,11 +28,11 @@ export const NavItem: React.FC<NavItemProps> = ({
       onMouseEnter={() => toggleHover(true)}
       onMouseLeave={() => toggleHover(false)}
     >
-      <TextLink href={href} color={hoverColor}>
+      <Link href={href} color={hoverColor} underlineBehavior={"none" as any}>
```

Comment:
The `as any` will be fixed once https://github.com/artsy/palette/pull/315 is merged